scripts/aws/setupdb.sh

```diff
@@ -17,7 +17,12 @@ psql -c "CREATE EXTENSION IF NOT EXISTS postgis;"
 # Run migrations
 envdir /etc/nyc-trees.d/env /opt/app/manage.py migrate
 # Load block face data
-envdir /etc/nyc-trees.d/env /opt/app/manage.py loaddata $DIR/../../src/nyc_trees/apps/survey/fixtures/blockface.json
+
+for i in `seq 1 7`;
```

Comment:
The `seq` part here can be done without a subshell:

```bash
for i in {1..7};
```

Alternatively, `ls` with wildcard matching can catch all of the `blockface_N.json` files:

```bash
ls /opt/app/apps/survey/fixtures/blockface_*.json
```

Combined with `xargs`, you can execute the management command for each match by piping the output:

```bash
ls /opt/app/apps/survey/fixtures/blockface_*.json | xargs -n1 envdir /etc/nyc-trees.d/env /opt/app/manage.py loaddata
```

The `-n1` argument just tells `xargs` to do one at a time vs. its default of all at once.